webapp/reviews/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+
+class Review(models.Model):
+    book = models.ForeignKey(
+        'book_details.Book',
+        related_name='reviews')
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='reviews')
+    anonymous = models.BooleanField(
+        'Anonymous')
```

Comment:
I believe this string is shown as help text to users; this should likely say something more along the lines of "Whether or not the comment was posted anonymously"